google-cloud-spanner/src/main/java/com/google/cloud/spanner/ForwardingResultSet.java

```diff
@@ -102,4 +103,9 @@ public ResultSetStats getStats() {
   public ResultSetMetadata getMetadata() {
     return delegate.get().getMetadata();
   }
+
+  @Override
+  public boolean initiateStreaming(AsyncResultSet.StreamMessageListener streamMessageListener) {
```

Comment:
nit: make it package-private (if possible), and otherwise mark with `@InternalApi`

Reply:
done